src/features/campaigns/components/ActivitiesOverview/items/EmailOverviewListItem.tsx

```diff
@@ -15,11 +15,17 @@ const EmailOverviewListItem: FC<EmailOverviewListItemProps> = ({
   focusDate,
 }) => {
   const email = activity.data;
-  const { data: emailStats } = useEmailStats(email.organization.id, email.id);
+  const { data: emailStats, lockedTargets } = useEmailStats(
```

Comment:
Same thing here, i.e. would it be possible to use the separate properties instead of `data`?